html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<meta charset="UTF-8">
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+<h1>Coding Form!</h1>
+<form action="">
+  <label for="email">Email
+    <div>
+      <input type="text" name="email">
+    </div>
+  </label>
+  <label for="reasons">Why is coding cool?
+    <div>
+      <select id="reasons" name="reasons">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+  </label>
+  <label for="language"><p>Favorite Language (choose one)</p>
```

Comment:
nice work here, the only thing to update is that each of the radio inputs needs its own label, you can wrap the inputs like this also: 
```html
      <label>
        <input type="radio" name="favoriteLanguage" value="JavaScript">
        JavaScript
      </label>
```

Reply:
![image](https://user-images.githubusercontent.com/96906988/151490522-5c52cbfc-e466-4e37-95ae-7b02999c4c8a.png)